src/components/Book/Book.css

```diff
@@ -0,0 +1,155 @@
+
+.percent-complete,
+.completed {
+  font-family: "Montserrat", sans-serif;
+}
+
+.percent-complete {
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  font-size: 0.875rem;
+  opacity: 0.5;
+  color: #121212;
+}
+
+.circular-progress-container {
+  width: 5.625rem;
+  height: 5.625rem;
+  padding: 0.63rem 0.63rem 0.75rem 0.75rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.circular-progress {
+  position: absolute;
+  border: solid 5px #307bbe;
+  border-left-color: #e8e8e8;
+  border-radius: 50%;
+  width: 4.3rem;
+  height: 4.3rem;
+  transform: rotate(45deg);
+}
+
+.books {
```

Comment:
- [ ] Good job so far 👏 👏 👏 You've done great job! However, there is a little suggestion I'd like to give. Please look at the alignment of the content nearby circular progress and try to align them as on the design given. I think it will be understood better if you see it in the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/230483446-0030befb-4f09-43f0-a554-02e81ea94e3b.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/230483529-3a36b012-8cff-4321-b512-838879bbec58.png)

Please move "completed" text slightly up so as to match the design given 🏃